packages/core/test/buttons/buttonTests.tsx

```diff
@@ -9,66 +9,83 @@ import { assert } from "chai";
 import { mount, shallow } from "enzyme";
 import * as React from "react";
 
+import * as Keys from "../../src/common/keys";
 import { AnchorButton, Button, Classes, IButtonProps, Spinner } from "../../src/index";
 
 describe("Buttons:", () => {
     buttonTestSuite(Button, "button");
-    buttonTestSuite(AnchorButton, "a");
+    buttonTestSuite(AnchorButton, "a", () => {
+        it("calls onClick when enter key pressed", () => {
```

Comment:
test these for `Button` too!
if it doesn't work identically then you've got more work to do!